TutorialApplication/MainViewController.m

```diff
@@ -0,0 +1,137 @@
+//
+//  MainViewController.m
+//  TutorialApplication
+//
+//  Created by Aleksandr Yegorov on 1/20/17.
+//  Copyright © 2017 Aleksandr Yegorov. All rights reserved.
+//
+
+#import "MainViewController.h"
+#import "Employee.h"
+#import "Organization.h"
+#import "DetailViewController.h"
+
+@interface MainViewController ()
+
+@property(strong, nonatomic) Organization *organization;
+
+@end
+
+@implementation MainViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    // Uncomment the following line to preserve selection between presentations.
+    // self.clearsSelectionOnViewWillAppear = NO;
+    
+    // Uncomment the following line to display an Edit button in the navigation bar for this view controller.
+    // self.navigationItem.rightBarButtonItem = self.editButtonItem;
+    Employee *emp1 = [[Employee alloc] initWithName:@"Stepka" lastName:@"Pupkin" salary:3000];
+    
+    self.organization =[[Organization alloc] initWithName:@"TheBestOrganization"];
+    
+    [self.organization addEmployee:emp1];
+    [self.organization addEmployeeWithName:@"Alex Yegorov"];
+    [self.organization addEmployeeWithName:@"Vova Kynovskiy"];
+    [self.organization addEmployeeWithName:@"Misha Davidayn"];
+    
+    NSLog(@"%@", [self.organization description]);
```

Comment:
To access properties use dot-notation, so it would be self.organization.description